### ISSUE TITLE
problem compiling Example 1 using Visual Studio 2010

### ISSUE THREAD

**Issue by jledwards666** [InductiveComputerScience/pbPlots#26]
I am a relatively new C programmer and am trying to run the pbPlots example 1 under the old (and free) Visual Studio 2010 C IDE.  If I am interpretting the error messages correctly, it appears that pbPlots' source file "supportlib.c" makes use of the "FILE" type in a manner incompatible with how its defined in VS2010's header file "stdio.h".  I am trying to include a screen grab (with the offending complaint highlighted in yellow) along with this Issue here but I'm not sure I attached it herein correctly... we'll see.

Bottom line, I have no idea whatsoever as to how to even approach this hiccup.  Thoughts/suggestions?

![tmp 20230509-192513](https://github.com/InductiveComputerScience/pbPlots/assets/133054431/2b915475-4d68-451e-b674-966dbae24c36)

**Comment by martinfjohansen** [comment]
Hi, could you try to increase the standard level of C to C99?
